packages/frontend/src/features/sync/components/SyncModalForm.tsx

```diff
@@ -191,6 +210,31 @@ export const SyncModalForm: FC<{ chartUuid: string }> = ({ chartUuid }) => {
 
                     <SelectGoogleSheetButton />
 
+                    <Group>
+                        <Switch
+                            label="Save in a new tab"
+                            checked={saveInNewTab}
+                            onChange={() => setSaveInNewTab(!saveInNewTab)}
+                        ></Switch>
+                        <Tooltip
+                            label={`Type a tab name to save the sync in, insead of overriding the first existing tab in the Google sheet.
+                                    This will create a new tab if it doesn't exist. We will still create a tab called metadata with the Sync information.`}
```

Comment:
Should we add a validation so they can't type `metadata`? I assume it's going to add data to metadata tab and then override the data with the real etadata.